src/pages/gateway/overview.md

```diff
@@ -32,6 +32,18 @@ The API Mesh enables developers to integrate a GraphQL Mesh or third-party APIs
 
 Adobe Commerce users can consider the API Mesh as a low-code way to integrate other APIs to the Storefront and Storefront Management APIs. It also provides a way for the Storefront API and the Storefront Management APIs to communicate with each other.
 
+## Security
+
+The GraphQL data plane used by API Mesh is public, but requires both a MeshID and an API Key to access. We recommend storing your API Key in a header, instead of using the API Key attached to your mesh's url.
+
+Basic authentication and authorization are provided by your Mesh ID and API Key. Additionally, API Mesh honors any downstream authorization headers provided by your [sources](source-handlers.md).
```

Comment:
Thanks for the write-up Jared.
I don't this basic authentication is provided by API Key. Mike can confirm.